exonum-java-binding-fakes/src/test/java/com/exonum/binding/fakes/services/transactions/ContextUtils.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2018 The Exonum Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.exonum.binding.fakes.services.transactions;
+
+import com.exonum.binding.common.crypto.PublicKey;
+import com.exonum.binding.common.hash.HashCode;
+import com.exonum.binding.storage.database.Fork;
+import com.exonum.binding.transaction.TransactionContext;
+
+/**
+ * Helper class to provide preconfigured context with default values.
+ */
+public final class ContextUtils {
```

Comment:
@dmitry-timofeev @bullet-tooth If we plan to create a separate test kit project in future, would it make sense to make a wiki page (or add some kind of TODO in code) with all the classes that would go into that test kit project? It seems that this one (and its two copies in cryptocurrency demo and QA service) would fit there.

Reply:
Are you thinking of documenting the prospective functionality of it? A wiki page sounds good for this, it then might evolve into the requirements.